device/network/network-iOS.js

```diff
@@ -109,7 +111,30 @@ Network.createNotifier = function(params) {
 
     //Android specs
     self.android = {};
-    self.android.isInitialStickyNotification = () => {};
+    self.android.isInitialStickyNotification = () => { };
+}
+
+Network.isConnected = function () {
```

Comment:
Duplicate

Reply:
I didn't understand which function is duplicated? @ozcanovunc

Reply:
Alright, I understand now. I changed it, could you check it? @ozcanovunc